src/platform/Linux/NFCCommissioningManagerImpl.cpp

```diff
@@ -610,34 +611,43 @@ CHIP_ERROR NFCCommissioningManagerImpl::SendToNfcTag(const Transport::PeerAddres
     NFCMessage * nfcMessage = new (std::nothrow) NFCMessage(pTargetedTagInstance, std::move(msgBuf));
     VerifyOrReturnLogError(((nfcMessage != nullptr) && (nfcMessage->GetMessage() != nullptr)), CHIP_ERROR_NO_MEMORY);
```

Comment:
if GetMessage is nullptr but nfcMessage is not null, we seem to leak the NFCMessage (very very unlikely though...).